ada/views.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import HttpResponseRedirect, render
+
+from .models import RedirectUrl, RedirectUrlsEntry
+
+
+def clickActivity(request: HttpRequest, pk: int, whatsappid: int) -> HttpResponse:
+    try:
+        redirect_url = RedirectUrl.objects.get(id=pk)
+    except (ValueError, RedirectUrl.DoesNotExist):
+        return render(request, "index.html", {"error": "HTTP Bad Request"})
+    store_url_entry = RedirectUrlsEntry(url=redirect_url)
+    store_url_entry.save()
+    destination_url = f"{redirect_url.symptom_check_url}" f"/?whatsappid={whatsappid}"
```

Comment:
It seems odd that we're adding a `/` here, what if it URL already has a `/`?

We should also use https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote to create the querystring, to ensure that it's a valid querystring.